.devcontainer/devcontainer.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "Primer Components",
+  "image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:12",
+  "forwardPorts": [8000],
+  "postCreateCommand": "pushd docs && yarn && popd && yarn"
```

```suggestion
  "postCreateCommand": "/bin/zsh -c 'pushd docs && yarn && popd && yarn'"
```

Looks like the [default shell for Docker operations is `/bin/sh`](https://docs.docker.com/engine/reference/builder/#run), which doesn't define `pushd`. If I try `pushd docs 2> error && yarn && popd && yarn` and inspect `error`, I see `/bin/sh: 1: pushd: not found`.